client/src/pages/private/UserView.js

```diff
@@ -220,6 +223,39 @@ export default () => {
                   }}
                 />
               </Box>}
+            {(values.role === 'employer') && <Box mt={3}>
```

Comment:
Isn't this condition already handled on line 204? I'm confused why we check for employer twice

Reply:
Good catch! This was some redundant code that I was going to use as a template and then forgot to remove it.